homeassistant/config_entries.py

```diff
@@ -265,9 +266,35 @@ class ConfigFlowResult(FlowResult, total=False):
 
     minor_version: int
     options: Mapping[str, Any]
+    subentries: Mapping[str, ConfigSubentryData]
     version: int
 
 
+class ConfigSubentryData(TypedDict):
```

Comment:
Should it be SubEntry or Subentry?

Reply:
It should be subentry imo: https://www.merriam-webster.com/dictionary/subentry#:~:text=Kids%20Definition-,subentry,under%20a%20more%20general%20entry